Core.Collectors/Collector/CollectorBase.cs

```diff
@@ -69,9 +69,9 @@ public async Task<bool> ProcessAsync(T collectionNode, long maxPageCount)
                 {
                     foreach (CollectionNode allowListCollectionNode in result.allowListStatus.Continuation(result.request))
                     {
-                        if (allowListCollectionNode is T)
+                        if (allowListCollectionNode is T t)
```

Comment:
Don't call variables 't', name something that would be easy to read e.g., castedAllowListCollectionNode or safeAllowlistCollectionNode (both are horrible, but I trust you can come up with a better naming).

Reply:
fixed as `typedAllowListCollectionNode`